core/templates/dev/head/pages/exploration_player/AudioBarDirective.js

```diff
@@ -28,19 +28,108 @@ oppia.directive('audioControls', [
       },
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/pages/exploration_player/' +
-        'audio_controls_directive.html'),
+        'audio_bar_directive.html'),
       controller: [
-        '$scope', 'AudioTranslationManagerService', 'AudioPlayerService',
-        'LanguageUtilService', 'AssetsBackendApiService',
+        '$scope', '$interval', 'AudioTranslationManagerService',
+        'AudioPlayerService', 'LanguageUtilService', 'AssetsBackendApiService',
         'AutogeneratedAudioPlayerService', 'PlayerPositionService',
+        'WindowDimensionsService', 'EVENT_ACTIVE_CARD_CHANGED',
         function(
-            $scope, AudioTranslationManagerService, AudioPlayerService,
-            LanguageUtilService, AssetsBackendApiService,
-            AutogeneratedAudioPlayerService, PlayerPositionService) {
+            $scope, $interval, AudioTranslationManagerService,
+            AudioPlayerService, LanguageUtilService, AssetsBackendApiService,
+            AutogeneratedAudioPlayerService, PlayerPositionService,
+            WindowDimensionsService, EVENT_ACTIVE_CARD_CHANGED) {
           // This ID is passed in to AudioPlayerService as a means of
           // distinguishing which audio directive is currently playing audio.
           var directiveId = Math.random().toString(36).substr(2, 10);
 
+          $scope.isAudioBarExpanded = false;
+          $scope.hasPressedPlayButtonOnce = false;
```

Comment:
I think I'm a bit confused about the semantics here -- this might merit a comment or a name change. In particular I can't tell whether this should be true or not if I click the Play button twice.

Reply:
changed to hasPressedPlayButtonAtLeastOnce